lib/runner/cli/clirunner.js

```diff
@@ -517,6 +517,10 @@ CliRunner.prototype = {
     if (this.argv.tag) {
       this.test_settings.tag_filter = this.argv.tag;
     }
+console.log('this.argv.tags', this.argv.tags);
```

Comment:
Thanks for pointing out, I had removed the console.log on a copy in my project, instead of the fork.